vscode-extension/package.json

```diff
@@ -32,8 +33,10 @@
     "vscode:prepublish": "npm run package",
     "build-templates": "cd ../loop-templates && npm ci && npm run build",
     "compile": "webpack",
+    "compile-web": "webpack",
```

Comment:
I feel like since there's no difference here, this command (and `watch-web`) aren't needed. But I do feel like you could restructure things to make it so you can build one or the other, or both.

I'm not great with Webpack, but something like this should work:
- Create a `webpack` dir in the root of the `vscode-extension` dir
- Inside the `webpack` dir, create two files - one named `webpack.main.js` and `webpack.web.js`
    - Move the main config object from `webpack.config.js` into `webpack.main.js`
    - Give each file a top-level `name` property of "main" and "web"
- Change `webpack.config.js` to just import those two files, and export them as a list
- Update `package.json` scripts to be:
```json
"compile": "webpack",
"compile:main": "webpack --config-name main",
"compile:web": "webpack --config-name web",
"watch:main": "webpack --config-name main --watch",
"watch:web": "webpack --config-name web --watch",
```
- You could also move any common things into a `webpack.common.js` file, and have the main and web files import and use the common things in their configs.

Hopefully that makes sense, and hopefully that actually works. Feel free to do whatever though!

Reply:
the compile-web was put there for testing purposes specific for vscode web extensions ([documentation](https://code.visualstudio.com/api/extension-guides/web-extensions)) but i agree since they are currently the same they can be cleaned up

Reply:
Gotcha. I **think** that documentation is just explaining that if you use `yo code` to generate the extension, it creates those scripts for you to build it. As far as I can tell, the only thing required is specifying the `"browser"` property in `package.json`. Otherwise, you can structure/build your project however you want. I was only suggesting the things in my last comment so that devs can build one or the other locally and actually **target** one or the other (rather than always compiling both). In the end, the only ones that _really_ matter are `"deploy"`, `"vscode:prepublish"`, and `"package"`, so anything else we provide are nice to have.

Reply:
yeah if dev wanted to update the task.json to match with the correct script thats totally doable and just up to the dev to change or add in to their local setup